src/App.tsx

```diff
@@ -1,14 +1,77 @@
 /* eslint-disable max-len */
-import React from 'react';
+import {
+  FC,
+  useEffect,
+  useState,
+  useCallback,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos, getUser } from './api';
+import { User } from './types/User';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [hasError, setHasError] = useState<boolean>(false);
+  const [isModalShown, setIsModalShown] = useState<boolean>(false);
```

Comment:
TS already knows about it, bc you put an initial value
```suggestion
  const [hasError, setHasError] = useState(false);
  const [isModalShown, setIsModalShown] = useState(false);
```